library/kicadLibrary/kicad_eSim-Library/eSim_Sources.lib

```diff
@@ -3,9 +3,9 @@ EESchema-LIBRARY Version 2.3
 #
 # AC
 #
-DEF eSim_AC v 0 40 Y Y 1 F N
+DEF AC v 0 40 Y Y 1 F N
```

Comment:
Any specific reason to rename it?

Reply:
'eSim_AC' was not acceptable by eSim in KicadtoNgspice step. Hence, converted to 'AC'.